diff --git a/target/linux/qualcommax/ipq50xx/config-default b/target/linux/qualcommax/ipq50xx/config-default
index 33e5611..5d6acb7 100644
--- a/target/linux/qualcommax/ipq50xx/config-default
+++ b/target/linux/qualcommax/ipq50xx/config-default
@@ -24,3 +24,7 @@ CONFIG_QCOM_Q6V5_MPD=y
 CONFIG_QCOM_Q6V5_WCSS_SEC=y
 CONFIG_QCOM_TMEL_QMP_MAILBOX=y
 CONFIG_SPI_QPIC_SNAND=y
+# CONFIG_ALLOC_SKB_PAGE_FRAG_DISABLE is not set
+# CONFIG_SKB_FAST_RECYCLABLE_DEBUG_ENABLE is not set
+CONFIG_SKB_RECYCLER=y
+CONFIG_SKB_RECYCLER_MULTI_CPU=y
